script/hello_name.py

```diff
@@ -4,9 +4,9 @@
 parser = argparse.ArgumentParser()
 parser.add_argument("--name", help="name the string you are use here")
 args = parser.parse_args()
-
-if args.name :
-  print('Hello', args.name)
-else :
-  args.name = input('Whats Your name?')
-  print('Hello', args.name)
\ No newline at end of file
+if args.name:
+ print('Hello', args.name)
```

Comment:
Please stick to the 2 spaces as indentation.

Reply:
Yes...I want to ask you about this, I cant use tab only spaces??

Reply:
It is always better to have spaces inside file.
But it is possible to use Tab on the keyboard as long as in you editor you have set the flag to convert tabs to spaces.